VSRAD.Package/BuildTools/Errors/Parser.cs

```diff
@@ -54,33 +63,47 @@ private static Message ParseClangMessage(string header)
         }
 
         private static readonly Regex ScriptErrorRegex = new Regex(
-            @"\*(?<kind>[EW]),(?<code>[^:(]+)(?>\s\((?<file>.+):(?<line>\d+)\))?:\s(?<text>.+)", RegexOptions.Compiled);
+            @"\*(?<kind>[EW]),(?<code>[^:(]+)(?>\s\((?<file>.+)\))?:\s(?<text>.+)", RegexOptions.Compiled);
 
-        private static readonly Regex ScriptErrorLocationInTextRegex = new Regex(
-            @"(?<text>.+)\s\((?<file>.+):(?<line>\d+)\)", RegexOptions.Compiled);
+        private static readonly Regex ScriptErrorTextRegex = new Regex(
+            @"(?<text>.+)\s\((?<file>.+)\)", RegexOptions.Compiled);
```

Comment:
Not clear naming for `file` group. It can be file with location or it can be simple error text because error like `W,undefined: undefined reference to 'printf' (some error text)` will also match the regex.